catalog/resource_metastore_data_access_test.go

```diff
@@ -169,56 +167,195 @@ func TestCreateDacWithGcpSA(t *testing.T) {
 		metastore_id = "abc"
 		name = "bcd"
 		is_default = true
-		gcp_service_account_key {
-			email = "a@example.com"
-			private_key_id = "b"
-			private_key = "abcdefg"
+		databricks_gcp_service_account {}
```

Comment:
Is this a breaking change? Can we provide compatibility in the meantime? Or is this just an oversight?

Reply:
the old syntax was just for private preview, and should not work anymore